pkg/builder/local_build_executor.go

```diff
@@ -152,6 +156,23 @@ func (be *localBuildExecutor) createOutputParentDirectory(inputRootDirectory fil
 	return d, nil
 }
 
+func (be *localBuildExecutor) createDev(inputRootDirectory BuildDirectory) error {
+	if err := inputRootDirectory.Mkdir("dev", 0777); err != nil && !os.IsExist(err) {
+		return util.StatusWrap(err, "Unable to create dev directory in input root")
+	}
+	devDir, err := inputRootDirectory.EnterDirectory("dev")
+	defer devDir.Close()
+	if err != nil {
+		return util.StatusWrap(err, "Unable to enter dev directory in input root")
+	}
+	for name, number := range be.inputRootCharacterDevices {
+		if err := devDir.Mknod(name, os.FileMode(os.ModeDevice|os.ModeCharDevice|0666), number); err != nil {
+			return util.StatusWrapf(err, "Mknod failed for device %#v", name)
```

Comment:
`"Failed to create character device %#v"`? 'Mknod' is a bit too technical here in my opinion.